misc.inc.php

```diff
@@ -923,16 +923,18 @@ function BuildCabinet($cabid,$face="front"){
 	$dev->ParentDevice=0;
 	$bounds=array(
 		'max'=>array('position'=>0,'height'=>0),
-		'min'=>array('position'=>0,'height'=>0),
+		'min'=>array('position'=>$cab->StartUNum,'height'=>0),
 	);
 
 	// Read in all the devices and make sure they fit the cabinet.  If not expand it
 	foreach($dev->Search() as $device){
-		if($device->Position==0){
```

Comment:
What about people that defined their device position to be -10 for some crazy reason, this is going to skip it and not adjust the rack height to start at -10